packages/blade/src/components/Chip/Chip.tsx

```diff
@@ -158,9 +177,12 @@ const _Chip: React.ForwardRefRenderFunction<BladeElementRef, ChipProps> = (
         onKeyDown={handleKeyboardPressedIn}
         onKeyUp={handleKeyboardPressedOut}
         inputProps={isReactNative() ? inputProps : {}}
-        style={{ cursor: _isDisabled ? 'not-allowed' : 'pointer' }}
+        style={{
+          cursor: _isDisabled ? 'not-allowed' : 'pointer',
+          ...selectorLabelStyles,
```

Comment:
Sorry for one more comment 🙈 But shouldn't we be adding width styles to outermost element? In this the basebox that we have? and let Chip take its width?

That way you also won't have to do `getBaseBoxStyles` hack since its BaseBox itself where you can directly pass width props.

Currently it feels too aggresive to add styles in style tag since they take very high priority. Plus I'm guessing it will break when I do `width={{ base: 'spacing.3', m: 'spacing.4' }}` since style tag won't support media query